### PR TITLE
Update staging site with MP5 zip; 22.0.0.1 SIG files

### PR DESCRIPTION
**Pull request by kinueng**
#### What was fixed?  (Issue # or description of fix)

Related to #2305

https://draft-openlibertyio.mybluemix.net/start/
image
image

#### Were the changes tested on
- [x] Firefox (Desktop)
- [x] Safari (Desktop)
- [x] Chrome (Desktop)
- [ ] Internet Explorer (Desktop)
- [ ] iOS (Mobile)
- [ ] Android (Mobile)
#### Running validation tools
- [x] https://validator.w3.org/checklink against https://draft-openlibertyio.mybluemix.net/start/
- [ ] https://validator.w3.org
- [x] IBM Equal Access Accessibility Checker
- [ ] Lighthouse (in Chrome dev tools)

### CODE REVIEW

File: src/main/content/_assets/js/builds.js [hunk 1]
```diff
@@ -88,6 +94,23 @@ function getAllowedBuilds(build_type, package_locations, liberty_version) {
     });
 }
 
+/**
+ * Return URL corresponding to the key
+ * @param {String} sig_key - e.g. "kernal.sig"
+ */
+function getURLForSig(list, sig_key) {
+    if(!list) {
+        return list;
```
Comment (steven1046) [inline src/main/content/_assets/js/builds.js]: It's a little unclear while looking at this function why we would want to return an empty value if nothing is passed in. Maybe if we add in the comments that it's optional.

File: src/main/content/_assets/js/builds.js [hunk 2]
```diff
@@ -266,58 +309,74 @@ function render_builds(builds, parent) {
...
-                                'ZIP</a></td>'
+                        var beta_zip_href = beta_package_locations[d].split('=')[1];
+                        //========== Get URL for the .sig file corresponding to the .zip file
+                        // The name has a lot of dots, so have to use lastIndexOf to separate
+                        // filename from file extension
+                        var beta_sig_name = beta_package_name.substring(0, beta_package_name.lastIndexOf('.'));
+                        var beta_sig_href = 
+                            getURLForSig(beta_package_sig_locs, beta_sig_name+'.sig');
+                        //========== Get URL for the .sha2 file
+                        var beta_sha2_href = ''; // TODO: Surface the href when DHE API has this data
```
Comment (steven1046) [inline src/main/content/_assets/js/builds.js:357]: Do we have an issue we can mark here for the TODO?

**Review by steven1046 — CHANGES_REQUESTED** [review]

File: src/main/content/_assets/js/builds.js (hunk 2 above)
Comment (kinueng) [reply src/main/content/_assets/js/builds.js:357]: I opened not to put the issue number in the comment.  The issue is 
 https://github.com/OpenLiberty/openliberty.io/issues/1734

File: src/main/content/_assets/js/builds.js (hunk 1 above)
Comment (kinueng) [reply src/main/content/_assets/js/builds.js]: Can you comment on if https://github.com/OpenLiberty/openliberty.io/pull/2441/commits/8d6b84c1bbc8298b50b75417ef9ba2f6014778cd has improved the readability of the code?

**Review by steven1046 — APPROVED** [review]

**Pull request merged**